### PR TITLE
Support Windows

### PR DESCRIPTION
**Pull request by mattn**
Just separated codes

### CODE REVIEW

**Comment by mattn** [comment]
Ah, should i make p-r from develop branch again?

**Comment by metalmatze** [comment]
Yep. This PR should definitely target `develop` :blush: :+1: 
Thanks for taking the time for this PR!

**Comment by mattn** [comment]
updated base of this p-r.

File: action/init.go [hunk 1]
```diff
@@ -5,9 +5,14 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/justwatchcom/gopass/gpg"
+	"github.com/mattn/go-colorable"
```
Comment (dominikschulz) [inline action/init.go]: I wonder why we'd need another color library along the existing one. If you believe your lib provides a benefit over the existing one feel free to open an separate PR to completely replace the old one, but for this PR I'd prefer if we could stick to the existing one.

File: fsutil/fsutil_windows.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		fmt.Printf("dir %s is a symlink. ignoring", path)
+		return false
+	}
+
+	return fi.Mode().IsRegular()
+}
+
+// Tempdir returns a temporary directory suiteable for sensitive data. On
+// Windows, just return empty string for ioutil.TempFile.
+func Tempdir() string {
```
Comment (dominikschulz) [inline fsutil/fsutil_windows.go]: The Tempdir function seems to be the only difference between fsutil_windows.go and fsutil_others.go.

Could you leave the common code in fsutil.go and only put the Tempdir function in fsutil_windows.go / fsutil_others.go?

**Comment by dominikschulz** [comment]
Very nice PR overall, thanks so much.

**Comment by mattn** [comment]
removed out variable. This was remaining to output color with color.GreenString(). It is not required by color.Green(). Also separated funcs in fsutil.

**Comment by brunoqc** [comment]
Any progress on this? I'm looking forward for this feature :).

**Comment by mattn** [comment]
This is already done. :)

**Comment by brunoqc** [comment]
> This is already done. :)

Can't wait for it to be merged then.

**Comment by mattn** [comment]
there are still some messages should be fixed. please wait

**Comment by mattn** [comment]
rebased to master

**Comment by dominikschulz** [comment]
Thanks! We'll try to get this one merged ASAP.

**Pull request merged**